tests/integration/study_data_blueprint/test_st_storage.py

```diff
@@ -238,7 +238,7 @@ def test_lifecycle__nominal(
 
         new_name = "Duplicate of Siemens"
         res = client.post(
-            f"/v1/studies/{study_id}/areas/{area_id}/storages/{siemens_battery_id}?new_cluster_name={new_name}",
+            f"/v1/studies/{study_id}/areas/{area_id}/st-storage/{siemens_battery_id}?new_cluster_name={new_name}",
```

Comment:
1. This will introduce a breaking change, the front-end must be also updated. Also think of others potential clients using the API (scripts team).
2. I don't think the naming for this signature is appropriate, RESTful API design principles advocate for using plural nouns to represent collections. The original endpoint `/storages` correctly uses the plural form, indicating a collection of storage resources.  Also abbreviations should be avoided for clarity purposes

You're right on the fact that we should make a clear dinstinction on the collections type here "short-term" for a type of storage.
> Note that potentially other types of storages will be added in the future, e.g. `long-term`

Suggestion:
Introduce Specific Storage Types as Sub-collections:
consider structuring the endpoints as follows to clearly indicate the storage type as part of the collection:

    `/studies/{uuid}/areas/{area_id}/storages/short-term/{storage_id}`
    `/studies/{uuid}/areas/{area_id}/storages/long-term/{storage_id}`

This approach maintains the plural form, adds clarity regarding the type of storage, and provides a scalable pattern for introducing additional storage types in the future.